src/pages/Features.js

```diff
@@ -0,0 +1,220 @@
+import tw from 'tailwind-styled-components'
+
+import SectionTitle from '../common/SectionTitle'
+
+import videoanimation from '../assets/videos/animation.mp4'
+import videoanimation2 from '../assets/videos/animation2.mp4'
+import features_1 from '../assets/images/features_1.png'
+import features_2 from '../assets/images/features_2.png'
+import features_3 from '../assets/images/features_3.png'
+
+const Video = tw.video`object-cover object-center bg-black`
+
+const Content = tw.div`px-6 md:px-12`
+const Intro = tw(
+  Content
+)`bg-gray-50 dark:bg-gray-950 text-center py-10 md:py-16`
+
+const Section = tw(Content)`py-4 md:py-10`
+const HSection = tw(Section)`md:grid md:grid-cols-2 space-x-2 max-w-6xl gap-x-12`
+
+const GridSection = tw.div`bg-gray-100 dark:bg-gray-950 py-4 px-8 rounded-lg`
+const GridSectionTitle = tw(SectionTitle)`text-left lg:text-center`
+
+const BulletList = tw.ul`list-disc p-4 md:p-0`
+const BulletListInside = tw.ul`list-disc list-inside p-4 md:p-0`
```

Comment:
I think we can extract a few common components from the above as they are mostly duplicated from the home page. Do you want to refactor it? Otherwise I'll try that after merging this PR.

Excellent work by the way. Thanks much! :)

Reply:
Yeah, I know the duplication issue :-). I prefer to commit it now and do the refactor after because I don't know so much about reactJs yet.

Reply:
Then let's just merge this for now :)

Reply:
Great, I commit it :-)